mimic/test/test_heat.py

```diff
@@ -159,6 +184,68 @@ def test_list_stacks_two(self):
         self.assertEqual(set(stack['stack_name'] for stack in two_stack_list),
                          set(['foostack', 'barstack']))
 
+    def test_check_stack(self):
+        """
+        Stack updates with correct check status.
+        """
+        foo_resp = self.create_stack('foostack')
+        self.check_stack('barstack', foo_resp['stack']['id'])
```

Comment:
Why is this done with `barstack` instead of `foostack`? I am guessing it should fail but thats not a concern since its not really checking.